### ISSUE TITLE
Add required `has` method

### ISSUE THREAD

**Issue by dio-az** [natesilva/keyv-anyredis#3]
KeyV interface now requires an additional [`has`](https://github.com/jaredwray/keyv/blob/e067b2c4b214351203dc2b1aa1278ddb1ab2c132/packages/keyv/src/index.js#L249-L261) method:

```
error TS2322: Type 'KeyvAnyRedis' is not assignable to type 'string | false | StorageAdapter | undefined'.
  Property 'has' is missing in type 'KeyvAnyRedis' but required in type 'Keyv<any>'.
```

**Comment by dio-az** [comment]
Hey @natesilva, did you have a chance to check this one?

**Comment by natesilva** [comment]
Thank you @diogoazevedos. This is published in version 3.2.0.
